paris/src/testDebug/java/com/airbnb/paris/utils/ShadowResourcesCompat.kt

```diff
@@ -0,0 +1,23 @@
+package com.airbnb.paris.utils
+
+import android.content.Context
+import android.graphics.Typeface
+import android.support.annotation.FontRes
+import android.support.v4.content.res.ResourcesCompat
+import org.robolectric.annotation.Implementation
+import org.robolectric.annotation.Implements
+
+@Implements(ResourcesCompat::class)
+class ShadowResourcesCompat {
+
+    companion object {
+
+        @JvmStatic
+        @Implementation
+        fun getFont(context: Context, @FontRes id: Int): Typeface {
+            return Typeface.create(id.toString(), Typeface.NORMAL)
```

Comment:
`id.toString()`, how is that working? Wouldn't `id` be an arbitrary `Int` here?

Reply:
Yes, this will create `Typeface` object with an arbitrary `Int` as a font family name. Since `Typeface` object equality check depends on font family name and style id, objects created from the same resource will be considered equal. And they won't clash with typefaces created from predefined names.